transfer/transfer-06-consumer-pull-http/README.md

```diff
@@ -0,0 +1,477 @@
+machines. The final goal of this example is to present the steps through which the 2 connectors will
+have to pass so that the consumer can have access to the data, held by the provider.
+
+Those steps are the following:
+
+* Running the provider connector
+* Running the consumer connector
+* Running a Http server that will receive the Endpoint Data Reference on the consumer side, that
+  contains the url to be used to get the data.
+* Register data plane instance for provider connector
+* Register data plane instance for consumer connector
+* Create an Asset on the provider (The asset will be the data to be shared)
+* Create an access policy on the provider (The policy will define the access right to the data)
+* Create a contract definition on the provider
+
+At this step, the connector should be able to fetch the catalog from the provider and to see the
+contract offer generated from the resources that have been created.
+
+Once the catalog is available, to access the data, the consumer should follow the following steps:
+
+* Performing a contract negotiation with the provider
+* Performing a transfer
+    * The consumer will initiate a file transfer
+    * The provider will send an EndpointDataReference to the consumer
+* The consumer could reach the endpoint and access the data
+
+Also, in order to keep things organized, the code in this example has been separated into several
+Java modules:
+
+* `connector`: contains the configuration and build files for both the
+  consumer and the provider connector
+* `backend-service`: represent the backend service where the consumer connector will send the
+  EndpointDataReference to access the data
+
+> For the sake of simplicity, we will use an in-memory catalog and pre-fill it with just one single
+> asset. This will be deleted after the provider shutdown.
+
+### Provider connector
+
+The provider connector is the one providing EndpointDataReference to the consumer after it initiates
+a transfer.
+
+### Consumer connector
+
+The consumer is the one "requesting" the data to the provider.
+
+# How to build a connector
+
+In fact, in the configuration of our example, both the provider and the consumer are connectors.
+Therefore, to set up our example, we need to start a connector with the configuration for a provider
+and another one with the configuration of a consumer.
+
+This section allows you to build the connector before launching it.
+
+```bash
+./gradlew transfer:transfer-06-consumer-pull-http:connector:build
+```
+
+After the build end you should verify that the connector jar is created in the directory
+[connector.jar](connector/build/libs/)
+
+# How to run a connector
+
+It is important to note that only the property file differs between the consumer and the supplier.
+You can find the configuration file in the directories below:
+
+* [provider](provider/provider-configuration.properties)
+* [consumer](consumer/consumer-configuration.properties)
```

Comment:
Could you add a few sentences explaining the properties that are new in this sample, like `edc.transfer.dataplane.sync.endpoint`?